modules/trycompile/src/Toolchain.cxx

```diff
@@ -48,8 +54,10 @@ CompilationResults compile_sketch(SketchSource src, stdfs::path prefix) {
     constexpr std::string_view silent_output = "null.txt"; // just in case
 #endif
 
-    const auto configure_cmd = fmt::format(R"(cmake -DSOURCE_FILE="{0}" -S "{1}" -B "{2}" > {3} && cmake --build "{4}" > {3} 2<&1)",
-                                           src.location.string(), share_dir.string(), build_dir.string(), silent_output, build_dir.string());
+
+    const auto configure_cmd = fmt::format(R"(cmake -DSOURCE_FILE="{0}" -DSOURCE_TYPE="{5}" -DARDUINOCLI_PATH="{6}" -S "{1}" -B "{2}" > {3} && cmake --build "{4}" > {3} 2<&1)",
```

Comment:
We might need 2<&1 on the first cmake call as well (I'm pretty sure it was there in the first place but got lost due to debugging stuff)